packages/commonwealth/server/routes/bulkBalances.ts

```diff
@@ -157,7 +164,7 @@ const bulkBalances = async (
     }
   }
 
-  return success(res, balances, bases);
+  return success(res, [balances, bases]);
```

Comment:
can we make this
```
return success(res, { balances, bases });
```
plz :)

Reply:
Also, this is the response I get:
![image](https://user-images.githubusercontent.com/33602208/191340234-699b2dcc-642b-4016-a079-619a396de7a4.png)
Can we get rid of the `base` key as well here